### ISSUE TITLE
Failure on Option 4 - Site Import

### ISSUE THREAD

**Issue by aitala** [psudug/nittany-vagrant#39]
Here is the error 'message' I'm getting...  the "rm: cannot remove `nittany/sites/default/settings.php': Permission denied"  is the where things start going wrong, but I also don't understand how it is happening...

---

nittany.sql

sent 30 bytes  received 7375667 bytes  1639043.78 bytes/sec
total size is 71434329  speedup is 9.69
rm: cannot remove `nittany/sites/default/settings.php': Permission denied
You are about to CREATE  the 'nittany' database. Do you want to continue? (y/n): y
Starting Drupal installation. This takes a few seconds ...                                                               [ok]
WD php: Exception: To start over, you must empty your existing database.To install to a different database, edit      [error]
the appropriate settings.php file in the sites folder.To upgrade an existing installation, proceed to the update
script.View your existing site. in install_begin_request() (line 301 of
/var/www/html/nittany/includes/install.core.inc).
WD php: Warning: Cannot modify header information - headers already sent by (output started at                      [warning]
/home/vagrant/.composer/vendor/drush/drush/includes/output.inc:38) in drupal_send_headers() (line 1212 of
/var/www/html/nittany/includes/bootstrap.inc).
Exception: <ul><li>To start over, you must empty your existing database.</li><li>To install to a different database, edit the appropriate <em>settings.php</em> file in the <em>sites</em> folder.</li><li>To upgrade an existing installation, proceed to the <a href="http://localhost/home/vagrant/.composer/vendor/drush/drush/update.php">update script</a>.</li><li>View your <a href="http://localhost/home/vagrant/.composer/vendor/drush/drush">existing site</a>.</li></ul> in install_begin_request() (line 301 of /var/www/html/nittany/includes/install.core.inc).
Drush command terminated abnormally due to an unrecoverable error.                                                    [error]
Do you really want to drop all tables in the database ipmsusa3_drupal7? (y/n): y
No tables to drop.                                                                                                       [ok]
ERROR 1045 (28000): Access denied for user 'ipmsusa3_d7'@'localhost' (using password: YES)
Query failed.                                                                                                         [error]
You need to specify an alias or run this command within a drupal site.                                                [error]
'drush' cache was cleared.                                                                                          [success]

The following post-sync operations will be done on the destination:
- Reset passwords and email addresses in user table
- Truncate Drupal's sessions table

Do you really want to sanitize the current database? (y/n):

**Comment by baf139** [comment]
Do you have a default settings file?

**Comment by aitala** [comment]
Yup... in both places.

**Comment by aitala** [comment]
Ok.. Oddness in the 'live' site.   Be careful of odd permissions/owners of the default directory and files within.

**Comment by btopro** [comment]
I don't think this is closed. Rsync can do some weird stuff w. User and perms or at least not what is expected. Maybe we could chown and chmod correctly after rsync to be safe, thus avoiding these issues.

**Comment by jpat14** [comment]
If you run sudo rsync, you can use -g to preserve group ownership and -o to preserve user ownership, and -p to preserve permissions (or archive mode performs all of these)

**Comment by btopro** [comment]
well, I think we just need to chown settings.php correctly to remove it and then we should make sure there's a default.settings.php in place. This is done so that the username / password are blown away from the theoretical production copy of settings.php and then rebuilt by vagrant via a drush SI minimal before doing the sql import.
